src/network/networkd-address.c

```diff
@@ -1407,6 +1407,51 @@ static int ipv4_dad_configure(Address *address) {
         return sd_ipv4acd_start(address->acd, true);
 }
 
+static int ipv4_dad_update_mac_one(Address *address) {
```

Comment:
It'd be nice to describe what the functional implications of this are (in the commit message).

Reply:
Done.